src/executables/util_service.c

```diff
@@ -1836,6 +1855,38 @@ process_server (int command_type, int argc, char **argv, bool show_usage, bool c
   return status;
 }
 
+/*
+ * process_server_start_single_node - start both a page and a transaction server,
+ *                                     the page server must be started before transaction server
+ *
+ * return:
```

Comment:
Remove comment lines that offer no information.
```suggestion
 *                                     the page server must be started before transaction server
```